packetbeat/etc/fields.yml

```diff
@@ -1288,6 +1288,74 @@ trans_event:
           description: >
             The cursor identifier returned in the OP_REPLY. This must be the value that was returned from the database.
 
+    - name: rpc
```

Comment:
Please run `make update` to re-generate the `packetbeat/docs/fields.asciidoc` document.